internal/cache/file/downloader/job_test.go

```diff
@@ -848,3 +848,26 @@ func (dt *downloaderTest) Test_validateCRC_ForTamperedFileWhenEnableCrcCheckIsFa
 	// Verify fileInfoCache update
 	dt.verifyFileInfoEntry(uint64(jobStatus.Offset))
 }
+
+func (dt *downloaderTest) Test_When_Parallel_Download_Is_Not_Enabled() {
+	objectName := "path/in/gcs/file1.txt"
+	objectSize := 8 * util.MiB
+	objectContent := testutil.GenerateRandomBytes(objectSize)
```

Comment:
you don't need to do dt.initiJobTest again as it was already done as part of test setup before every test.
(Also, you can explicitly set dt.job.fileCacheConfig.EnableParallelDownloads = false as you are doing in case of enabled)